.github/workflows/package-validation.yml

```diff
@@ -0,0 +1,27 @@
+name: Package Validation
+
+on:
+  pull_request:
+    branches: [ 'main*' ]
+    paths-ignore:
+    - '**.md'
+
+jobs:
+  build-test:
+    runs-on: windows-latest
+    env:
+      RunPackageValidation: true
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # fetching all
+
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v3
+
+    - name: Install dependencies
+      run: dotnet restore
+
+    - name: Pack
+      run: dotnet pack --configuration Release --no-restore
```

Comment:
Could you add two jobs for this workflow similar to the existing CI workflow? One for [stable releases](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/workflows/ci.yml#L14) and one for [experimental features](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/workflows/ci.yml#L42). You would have to split the build and pack steps for that.

Reply:
Thanks @utpilla! I just splitted the job as suggested.

Please let me know in case there's anything else.